### DISCUSSION TITLE
Question on export misorientation values

### DISCUSSION THREAD

**Discussion by RicCNR** [mtex-toolbox/mtex#65] [Ask Anything] [answered]
Dear Ralf,
thank you for your job.
I contact for the following issue:
i am working on misorientation analysis (both intergranular and boundary), wih the corresponding visualization and histogram distribution; and it is very interesting.
I am working with the points, for example:

``` matlab
mori = ebsd('Fo').mis2mean 
bnd_FoFo.misorientation.angle./degree
```

ecc..
About this I would like to study in depth the specific values, so  I ask you:
 is it possible to export the misorientation values (for both intergranular case and the boundary case) on that the aforesaid visualizations and distribution work, directly to an ASCII file? 
What can be the correct procedure for that?

Thank you Ralf

Regards

RicCNR

**Comment by kilir** [comment, marked as answer]
Hi RicCNR,
did you actually try export?
e.g. `export(mori,'output.txt')`, etc... .Otherwise you try csvwrite, for example `csvwrite('out.txt',bnd_FoFo.misorientation.angle./degree)`
Cheers,
Rüdiger

**Comment by RicCNR** [comment]
Hi Rudiger,
thanks a lot for your suggestion.
In fact I am already working on the first one `export(mori,'output.txt')`. 
But, the second command `csvwrite('out.txt',bnd_FoFo.misorientation.angle./degree)` is actually useful to obtain the misoritation values on that the visualization pixels work. 
I didn't remember it, so thank you again for your very useful indication

Regards

RicCNR
